README.md

```diff
@@ -5,19 +5,25 @@ Discord bot for recording & recalling quotes
 ## Setup
 
 - [Install Air](https://github.com/air-verse/air?tab=readme-ov-file#via-go-install-recommended).
+  - Ensure `air` is accessible from `$PATH`.
+  - It should be located in `~/go/bin` on Linux.
```

Comment:
```suggestion
  - It should be located in `~/go/bin` on Linux / Mac.
```

Reply:
I think this comment should just be removed entirely, setting up your Go install's path is a fairly standard part of setting up a new Go install, and once the next go release is out they'll be migrated to using Go tools anyway, at which point this should no longer be necessary